components/landing/LandingCta.vue

```diff
@@ -42,13 +46,13 @@ export default class extends Vue {
     width: 12rem;
   }
 
+  &:hover {
```

Comment:
@korgan00 would you mind to give it a try and implement the same effect for the `active` state?